extensions/metube/src/clipboard-to-metube.ts

```diff
@@ -0,0 +1,18 @@
+import { showHUD, Clipboard } from "@raycast/api";
+import { addToQueue } from "./helper.js";
+
+export default async function main() {
+  const clip = await Clipboard.readText();
+  if (clip) {
```

Comment:
Here you can also validate if it's even a link (and maybe if it's a youtube?)

Reply:
I could validate if it follows a generic link pattern, sure. However, MeTube works through yt-dlp which supports lots and lots of websites and not just YT, so I don't think limiting it to just YT would be helpful in this case but I still appreciate the suggestion.